lms/templates/admin/macros.html.jinja2

```diff
@@ -201,3 +201,33 @@
         </div>
     {% endcall %}
 {% endmacro %}
+{% macro tab_headers(request, tabs, active) %}
```

Comment:
This is probably the less nice part of this PR.

Having one route render one tab is pretty easy but this deals mostly with setting the "is-active" in the right tab.